File: app/frontend/ea-project/src/models/input/UploadUserPhotoInput.js
import BaseInput from "./BaseInput";

export default class UploadUserPhotoInput extends BaseInput {
  constructor(id = 0, photoBase64 = '', lazyLoad = false) {
    super(lazyLoad);
    this.id = id;
    this.photoBase64 = photoBase64;
  }
}

File: app/backend/src/main/java/eaproject/input/UploadUserPhotoInput.java
package eaproject.input;

import java.io.Serializable;

public class UploadUserPhotoInput extends BaseInput implements Serializable {
    private Integer id;
    private String photoBase64;

    private String fileName;

    private String contentType;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getPhotoBase64() {
        return photoBase64;
    }

    public void setPhotoBase64(String photoBase64) {
        this.photoBase64 = photoBase64;
    }

    public String getContentType() { return contentType; }

    public void setContentType(String contentType) { this.contentType = contentType; }

    public String getFileName() { return fileName; }

    public void setFileName(String fileName) { this.fileName = fileName; }
}
